drag_and_drop_v2/public/css/drag_and_drop_edit.css

```diff
@@ -187,11 +188,18 @@
     width: 35%;
 }
 
+.xblock--drag-and-drop--editor .items-form .zone-checkbox {
+    width: initial;
+}
+
 .xblock--drag-and-drop--editor .items-form .item-image-url {
     width: 81%;
     margin: 0 1%;
 }
 
+.xblock--drag-and-drop--editor .items-form .item-text {
+    margin: 0 1%;
+}
```

Comment:
@haikuginger DRY: You can remove this rule and add the selector to the rule in L181 above.